src/uu/tac/src/tac.rs

```diff
@@ -251,13 +252,15 @@ fn tac(filenames: &[&str], before: bool, regex: bool, separator: &str) -> UResul
         } else {
             let path = Path::new(filename);
             if path.is_dir() {
-                let e: Box<dyn UError> = TacError::InvalidArgument(String::from(filename)).into();
+                let e: Box<dyn UError> =
+                    TacError::InvalidArgument(filename.to_string_lossy().to_string()).into();
```

Comment:
No strong opinion, but in other cases, you've fixed the String type in the errors.